### PR TITLE
LPS-160061 Single Approver is not updated when User language is not English

### PR DESCRIPTION
**Pull request by seltong**
The frontend fix was sent [here](https://github.com/brianchandotcom/liferay-portal/pull/122274) and I'm sending the backend fix in this pull request.

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/123063

**Comment by seltong** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
@MarinhoFeliphe Please, remember to review this PR when possible.

File: modules/apps/portal-workflow/portal-workflow-kaleo-definition-impl/src/main/java/com/liferay/portal/workflow/kaleo/definition/internal/parser/BaseNodeValidator.java [hunk 1]
```diff
@@ -46,10 +48,27 @@ public void validate(Definition definition, T node)
...
+			for (Map.Entry<Locale, String> labelEntry : labelMap.entrySet()) {
+				String label = labelEntry.getValue();
+
+				if (label.length() > 200) {
+					throw new KaleoDefinitionValidationException.
+						MustSetValidNodeNameLength(200, label);
+				}
+			}
+		}
+	}
```
Comment (MarinhoFeliphe) [inline modules/apps/portal-workflow/portal-workflow-kaleo-definition-impl/src/main/java/com/liferay/portal/workflow/kaleo/definition/internal/parser/BaseNodeValidator.java:66]:
````
if (labelMap == null) {
  return;
}

for (Map.Entry<Locale, String> labelEntry : labelMap.entrySet()) {
  String label = labelEntry.getValue();
  
  if (label.length() > _MAX_TEXT_SIZE) {
	  throw new KaleoDefinitionValidationException.
		  MustSetValidNodeNameLength(_MAX_TEXT_SIZE, label);
  }
}
````

File: modules/apps/portal-workflow/portal-workflow-kaleo-definition-api/src/main/java/com/liferay/portal/workflow/kaleo/definition/Node.java [hunk 2]
```diff
@@ -79,7 +81,14 @@ public String getDefaultLabel() {
 			return _name;
 		}
 
-		return _labelMap.get(LocaleUtil.getDefault());
```
Comment (MarinhoFeliphe) [inline modules/apps/portal-workflow/portal-workflow-kaleo-definition-api/src/main/java/com/liferay/portal/workflow/kaleo/definition/Node.java:82]: User -> Site default -> Default -> _name

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by seltong** [comment]
ci:forward

**Pull request closed without merging**